xml/admin_ceph_libvirt.xml

```diff
@@ -251,15 +252,14 @@ Id    Name                           State
    <step>
     <para>
      If your &ceph; cluster has authentication enabled (it does by default),
-     you must generate a secret.
+     you must generate a secret. Open an editor of your choice and create
+     a file called <filename>secret.xml</filename> with the following content:
     </para>
-<screen>cat &gt; secret.xml &lt;&lt;EOF
```

Comment:
Only experienced command line users know that they need to copy the whole code in the screen box